himportscan/src/HImportScan/ImportScanner.hs

```diff
@@ -92,10 +92,8 @@ scanImports dynFlags filePath contents = do
         map GHC.unLoc $
           GHC.getOptions dynFlagsWithExtensions sb filePath
   GHC.getImports dynFlagsWithExtensions sb filePath filePath >>= \case
-    -- It's important that we error in this case, as otherwise the parser has gone terribly wrong probably.
-    -- It's also ok to print the error, as it is usually descriptive and well formatted.
-    -- The way we error here is that we're passing unexpected output to the go library.
-    -- This is far from ideal, however handling this better would require being able to communicate errors better to go.
```

Comment:
There's is no interference with the parsing of json because gazelle reads stdout, while these errors should go to stderr. Also, I think gazelle will fail if `himportscan` ends with a non-zero exit code, which should be the case when dying with an exception.

Reply:
If by `gazelle` you mean `gazelle_haskelL_modules`, it reads both stdout and stderr via https://pkg.go.dev/os/exec#Cmd.CombinedOutput

Reply:
Good catch, it would make sense to handle the outputs separately.

Reply:
I tried to handle this in https://github.com/tweag/gazelle_haskell_modules/pull/32/commits/55caa5d865b62e5a38097b8b2d863de1c9ede911

Reply:
Thanks!